pkg/cmd/pipelinerun/logs.go

```diff
@@ -109,9 +113,33 @@ func Run(opts *options.LogOptions) error {
 
 	log.NewWriter(log.LogTypePipeline, opts.Prefixing).Write(opts.Stream, logC, errC)
 
+	// get pipelinerun status
+	if opts.ExitWithPrError {
+		clients, err := opts.Params.Clients()
+		if err != nil {
+			return err
+		}
+		ctx := context.Background()
+		pr, err := clients.Tekton.TektonV1().PipelineRuns(opts.Params.Namespace()).Get(ctx, opts.PipelineRunName, metav1.GetOptions{})
```

Comment:
better to use get function from pipelinerunpkg which uses dynamic client, like you can do `pipelinerunpkg.GetPipelineRun(pipelineRunGroupResource, clients, opts.PipelineRunName, opts.Params.Namespace())`